Feature/homework 10 1

mrDrauzer opened the pull request.

mrDrauzer committed:
Изменен импорт с import mask  на from masks import get_mask_account, get_mask_card_number. Изменены аргументы в связи с правкой.

mrDrauzer committed:
Добавлен модуль processing с функциями
 1. filter_by_state
     Принимает список словарей и опционально значение для ключа state (по умолчанию 'EXECUTED'). Функция возвращает новый список словарей, содержащий только те словари, у которых ключ state соответствует указанному значению.

 2. sort_by_date
    Принимает список словарей и необязательный параметр, задающий порядок сортировки (по умолчанию — убывание). Функция возвращает новый список, отсортированный по дате (date).

mrDrauzer committed:
Добавлено описания к функциям модуля processing
 1. filter_by_state
     Принимает список словарей и опционально значение для ключа state (по умолчанию 'EXECUTED'). Функция возвращает новый список словарей, содержащий только те словари, у которых ключ state соответствует указанному значению.

 2. sort_by_date
    Принимает список словарей и необязательный параметр, задающий порядок сортировки (по умолчанию — убывание). Функция возвращает новый список, отсортированный по дате (date).

mrDrauzer committed:
Добавлено описания к функциям модуля processing
 1. filter_by_state
     Принимает список словарей и опционально значение для ключа state (по умолчанию 'EXECUTED'). Функция возвращает новый список словарей, содержащий только те словари, у которых ключ state соответствует указанному значению.

 2. sort_by_date
    Принимает список словарей и необязательный параметр, задающий порядок сортировки (по умолчанию — убывание). Функция возвращает новый список, отсортированный по дате (date).

mrDrauzer committed:
Create README.md

Данный проект является учебным На данном жтапе реализованы функции скрытия части номера карты либо счета. Добавлена функция сортировки проводок по дате и создания списка выполненых либо не выполненых проводок.

mrDrauzer commented:
изминения в соответствии с коментариями

painassasin commented on src/processing.py:
```diff
@@ -0,0 +1,37 @@
+    return sort_by_date
+
+
+list_pr = [
```
Весь исполняемый код из модулей нужно убрать, иначе нельзя будет импортировать функции написанные в нем (весь код ниже)

painassasin commented on src/processing.py:
```diff
@@ -0,0 +1,37 @@
+                list_operations_state.append(operations)
+        return list_operations_state
+
+    except Exception:
```
Тут ошибки быть не должно, а если она и есть, то ее не нужно перехватывать

mrDrauzer committed:
Добавлено описания к функциям модуля processing
 1. filter_by_state
     Принимает список словарей и опционально значение для ключа state (по умолчанию 'EXECUTED'). Функция возвращает новый список словарей, содержащий только те словари, у которых ключ state соответствует указанному значению.

 2. sort_by_date
    Принимает список словарей и необязательный параметр, задающий порядок сортировки (по умолчанию — убывание). Функция возвращает новый список, отсортированный по дате (date).

mrDrauzer committed:
Merge remote-tracking branch 'origin/feature/homework_10_1' into feature/homework_10_1

mrDrauzer committed:
Добавлено описания к функциям модуля processing
 1. filter_by_state
     Принимает список словарей и опционально значение для ключа state (по умолчанию 'EXECUTED'). Функция возвращает новый список словарей, содержащий только те словари, у которых ключ state соответствует указанному значению.

 2. sort_by_date
    Принимает список словарей и необязательный параметр, задающий порядок сортировки (по умолчанию — убывание). Функция возвращает новый список, отсортированный по дате (date).

mrDrauzer commented:
Вроде все поправил.
